### PR TITLE
Lfl loom push notification data

### PR DESCRIPTION
**Pull request by LukeLamothe**

### CODE REVIEW

**Comment by LukeLamothe** [comment]
Holy crap it built...

File: application/android/src/co/theengine/loomdemo/LoomMobile.java [hunk 1]
```diff
@@ -174,14 +186,80 @@ public static boolean openedWithCustomScheme()
...
+    ///processes the notification data from our custom handler in LoomCustomNotificationReceiver
+    public static void processNotificationData(Bundle bundle)
+    {
+        if(bundle == null)
+        {
+            return;
+        }
+
+        //store the notification data as our custom payload
+        String message = bundle.getString("com.parse.Data");
```
Comment (bengarney) [inline application/android/src/co/theengine/loomdemo/LoomMobile.java]: Should this be configurable?

**Comment by bengarney** [comment]
One nit!

**Comment by LukeLamothe** [comment]
You mean com.parse.data?

Nope, that's a built in Parse set key that cannot be changed anywhere. The
entire system is built around Parse remote notifications, specifically the
Android code, so there is no easy way to say, bind it to Urban Airship or
custom PNs.

**Comment by bengarney** [comment]
OK, sounds good. I am landing this.

**Pull request merged**